packages/hir/src/lowering/packages.candy

```diff
@@ -0,0 +1,22 @@
+use compiler_utils
+use incremental
+
+fun getDirectDependencies(context: QueryContext<List<CompilerError>>, package: Package): Set<Package> {
+  // Returns all direct dependencies of the given `package` as declared in the `.candyspec`.
+
+  todo("getDirectDependencies")
```

Comment:
This would just be `package.getCandyspec(context).dependencies` (`compiler_utils/src/candyspec.candy`) – do we actually need that as a separate method?